rust/libnewsboat/src/cliargsparser.rs

```diff
@@ -842,8 +842,8 @@ mod tests {
     }
 
     #[test]
-    fn t_sets_display_msg_and_asks_to_exit_with_failure_if_argument_to_dash_l_is_outside_1_to_6_range(
-    ) {
+    fn t_sets_display_msg_and_asks_to_exit_with_failure_if_argument_to_dash_l_is_outside_1_to_6_range()
+     {
```

Comment:
The extra space before the brace irks me. Would the new rustfmt accept the following?

```suggestion
    fn t_sets_display_msg_and_asks_to_exit_with_failure_if_argument_to_dash_l_is_outside_1_to_6_range()
    {
```

There a bunch more places with exact same edit, below.

Reply:
It looks like rustfmt will add back the space in all of the 4 occurences I found.
I don't see a rustfmt setting which directly controls this whitespace.

Only ways I know how to avoid this whitespace:
- Reduce length of these test names
- Increase max line lenth using rustfmt's [`max_width` setting](https://rust-lang.github.io/rustfmt/?version=master&search=#max_width)
However, increasing the max line length will also change a bunch of chained method calls.
That might actually look nicer, but I will leave it to you to decide if you want this.
See https://github.com/dennisschagt/newsboat/commit/0121e0705d8a0143868c22d8eb801cc89b6786ff for changes in formatting if max line length is changed to 120 (from default 100)